client/my-sites/activity/activity-log/index.jsx

```diff
@@ -320,7 +320,12 @@ class ActivityLog extends Component {
 				<Fragment>
 					{ config.isEnabled( 'activity-filterbar' ) &&
 						! isFilterEmpty && <Filterbar siteId={ siteId } filter={ filter } /> }
-					<EmptyContent title={ translate( 'No matching events found.' ) } />
+					<EmptyContent
+						title={ translate( 'No matching events found.' ) }
+						line={ translate( 'Try adjusting your date range or acitvity type filters' ) }
+						action={ translate( 'Remove all filters' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **9** times:
`translate( 'Remove All Filters' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).